pkg/resource/deploy/plan_executor.go

```diff
@@ -38,12 +38,13 @@ type planExecutor struct {
 }
 
 // execError creates an error appropriate for returning from planExecutor.Execute.
-func execError(message string, preview bool) error {
+func (pe *planExecutor) reportExecResult(message string, preview bool) {
```

Comment:
can you update the comment on this function?

Reply:
Sure.  Will do in followup PR.